netket/graph/hypergrid.py

```diff
@@ -0,0 +1,132 @@
+from graph import NetworkX
+
+import numpy as _np
+import networkx as _nx
+
+
+class Hypergrid(NetworkX):
```

Comment:
Can we just call it `Grid`? I think there is no need to add "hyper", a grid can be any dimension. (For `Hypercube` it makes sense, as cube by itself is 3d.)

Reply:
I named it `Grid` now